### PR TITLE
Fix #1560: Create Operation does not return proximity OTP

### LINKED ISSUE #1560: Create Operation does not return proximity OTP
Even if the create operation request contains `proximityCheckEnabled=true`, the response is missing the proximity OTP.

### PR DESCRIPTION
**Pull request by banterCZ**

### CODE REVIEW

File: powerauth-java-server/src/test/java/io/getlime/security/powerauth/app/server/service/behavior/tasks/OperationServiceBehaviorTest.java [hunk 1]
```diff
@@ -89,8 +89,27 @@ void testCreateOperationWithActivationId() throws Exception {
 
         final OperationDetailResponse operationDetailResponse = operationService.createOperation(request);
         final OperationEntity savedEntity = operationRepository.findOperation(operationDetailResponse.getId()).get();
+
+        assertTrue(operationRepository.findOperation(operationDetailResponse.getId()).isPresent());
+        assertEquals(ACTIVATION_ID, savedEntity.getActivationId());
+        assertNull(operationDetailResponse.getProximityOtp());
```
Comment (zcgandcomp) [inline powerauth-java-server/src/test/java/io/getlime/security/powerauth/app/server/service/behavior/tasks/OperationServiceBehaviorTest.java:95]: Assert to null is true IFF the template itself is not parameterized for PAC. This is a little bit strange. but overall acceptable.

File: powerauth-java-server/src/test/java/io/getlime/security/powerauth/app/server/service/behavior/tasks/OperationServiceBehaviorTest.java [hunk 2]
```diff
@@ -89,8 +89,27 @@ void testCreateOperationWithActivationId() throws Exception {
...
+        request.setActivationId(ACTIVATION_ID);
+        request.setApplications(List.of(APP_ID));
+        request.setTemplateName("test-template");
+        request.setUserId(USER_ID);
+        request.setProximityCheckEnabled(true);
+
+        final OperationDetailResponse operationDetailResponse = operationService.createOperation(request);
+        final OperationEntity savedEntity = operationRepository.findOperation(operationDetailResponse.getId()).get();
+
         assertTrue(operationRepository.findOperation(operationDetailResponse.getId()).isPresent());
```
Comment (zcgandcomp) [inline powerauth-java-server/src/test/java/io/getlime/security/powerauth/app/server/service/behavior/tasks/OperationServiceBehaviorTest.java:110]: Using `operationService.operationDetail` gives us better coverage than just an operationRepository method.

**Review by zcgandcomp — CHANGES_REQUESTED** [review]
Sorry, can we make a separate test? Now the asserts are in the method testing `activationId` attribute.

File: powerauth-java-server/src/test/java/io/getlime/security/powerauth/app/server/service/behavior/tasks/OperationServiceBehaviorTest.java (hunk 1 above)
Comment (banterCZ) [reply powerauth-java-server/src/test/java/io/getlime/security/powerauth/app/server/service/behavior/tasks/OperationServiceBehaviorTest.java:95]: 🤔 true, I will add implicit and explicit proximity check enables.

File: powerauth-java-server/src/test/java/io/getlime/security/powerauth/app/server/service/behavior/tasks/OperationServiceBehaviorTest.java (hunk 2 above)
Comment (banterCZ) [reply powerauth-java-server/src/test/java/io/getlime/security/powerauth/app/server/service/behavior/tasks/OperationServiceBehaviorTest.java:110]: Ok, why not. It is a unit test, but `#operationDetail` is not covered yet, so it is an improvement.

**Comment by banterCZ** [comment]
> Sorry, can we make a separate test? Now the asserts are in the method testing `activationId` attribute.

I have created a separate test method. But ok, let's test it without `activationId`.

**Review by romanstrobl — APPROVED** [review]
Looks OK.

File: powerauth-java-server/src/test/java/io/getlime/security/powerauth/app/server/service/behavior/tasks/OperationServiceBehaviorTest.java [hunk 3]
```diff
@@ -89,8 +89,47 @@ void testCreateOperationWithActivationId() throws Exception {
...
+        request.setTemplateName("test-template-proximity-check");
+        request.setApplications(List.of(APP_ID));
+        request.setUserId("test-user");
+
+        final OperationDetailResponse operationDetailResponse = operationService.createOperation(request);
+        assertNotNull(operationDetailResponse.getProximityOtp());
+
+        final OperationDetailRequest detailRequest = new OperationDetailRequest();
+        detailRequest.setOperationId(operationDetailResponse.getId());
+        assertNotNull(operationService.operationDetail(detailRequest));
```
Comment (zcgandcomp) [inline powerauth-java-server/src/test/java/io/getlime/security/powerauth/app/server/service/behavior/tasks/OperationServiceBehaviorTest.java:129]:
```suggestion
        final OperationDetailRequest detailRequest = new OperationDetailRequest();

        final OperationDetailResponse operationDetail = operationService.operationDetail(detailRequest);
        assertNotNull(operationDetail);
        assertNotNull(operationDetail.getProximityOtp());
        assertNull(operationDetail.getActivationId());
        ```

**Review by zcgandcomp — CHANGES_REQUESTED** [review]

**Review by zcgandcomp — APPROVED** [review]
Looks OK.

**Pull request merged**